solr.sh

```diff
@@ -64,5 +67,10 @@ then
   SOLR_PORT="8080"
 fi
 
+if [ -z "$SOLR_ADDITIONAL_OPTIONS" ]
+then
+  SOLR_ADDITIONAL_OPTIONS=""
+fi
+
 export SOLR_LOGS_DIR=$SOLR_LOGS_DIR
-$SOLR_BIN/solr $1 -p $SOLR_PORT -s $SOLR_HOME -m $SOLR_HEAP -a "-Dsolr.log=$SOLR_LOGS_DIR"
+$SOLR_BIN/solr $1 -p $SOLR_PORT -s $SOLR_HOME -m $SOLR_HEAP -a "-Dsolr.log=$SOLR_LOGS_DIR $SOLR_ADDITIONAL_OPTIONS"
```

Comment:
I notice that $SOLR_ADDITIONAL_OPTIONS is inside the quotes of the -Dsolr.log parameter. Is that intentional? I think it would probably be safer to have it outside those quotes... but let me know if you're having trouble getting it working that way. We also should add equivalent functionality to solr.bat for consistent Windows support. If you're not familiar with dealing with Windows batch files, I'll be happy to take care of that. I'm just going to add a TODO checkbox so we don't forget about it before merging.